### DISCUSSION TITLE
24/7

### DISCUSSION THREAD

**Discussion by Alfaboyratorr** [SudhanPlayz/Discord-MusicBot#887] [Q&A] [answered]
Its there any method to make the bot stay connected 24/7?

**Comment by DevAmiyo** [comment, marked as answer]
- If you're using the [`master`](https://github.com/SudhanPlayz/Discord-MusicBot/tree/master) branch, make sure that you set [`"24/7": true`](https://github.com/SudhanPlayz/Discord-MusicBot/blob/master/botconfig.js) in [botconfig.js](https://github.com/SudhanPlayz/Discord-MusicBot/blob/master/botconfig.js)
![24/7 botconfig.js](https://user-images.githubusercontent.com/84950599/173221525-e7b81a6c-4f4e-4c31-b324-42bc13feb68d.png)[^1]
- If you're using the [`v5`](https://github.com/SudhanPlayz/Discord-MusicBot/tree/v5) branch, well there's already a [`/247`](https://github.com/SudhanPlayz/Discord-MusicBot/blob/v5/commands/slash/247.js) command.

[^1]: Edit: Added image for better understanding

**Reply by Alfaboyratorr** [reply]
Thank you so much

**Reply by DevAmiyo** [reply]
You're welcome :D
